templates/partial/_navigation.html

```diff
@@ -0,0 +1,132 @@
+          <a class="p-navigation__link-anchor" href="#products-content" onfocus="fetchDropdown('/partial/_navigation-products', 'products-content');">Products</a>
+        </li>
+        <li class="p-navigation__dropdown-link {% if request.path|get_nav_path == 'partners' %}is-selected{% endif %}" role="menuitem" id="partners" onmouseover="fetchDropdown('/partial/_navigation-partners', 'partners-content'); this.onmouseover = null;">
+          <a class="p-navigation__link-anchor" href="#partners-content" onfocus="fetchDropdown('/partial/_navigation-partners', 'partners-content');">Partners</a>
+        </li>
+        <li class="p-navigation__dropdown-link {% if request.path|get_nav_path == 'careers' %}is-selected{% endif %}" role="menuitem" id="careers" onmouseover="fetchDropdown('/partial/_navigation-careers', 'careers-content'); this.onmouseover = null;">
+          <a class="p-navigation__link-anchor" href="#careers-content" onfocus="fetchDropdown('/partial/_navigation-careers', 'careers-content');">Careers</a>
+        </li>
+      </ul>
+      <noscript>
+        <ul class="p-navigation__links">
+          <li class="p-navigation__link {% if request.path|get_nav_path == 'products' %}is-selected{% endif %}" role="menuitem">
+            <a href="/products">Products</a>
+          </li>
+          <li class="p-navigation__link {% if request.path|get_nav_path == 'partners' %}is-selected{% endif %}" role="menuitem">
+            <a href="/partners">Partners</a>
+          </li>
+          <li class="p-navigation__link {% if request.path|get_nav_path == 'careers' %}is-selected{% endif %}" role="menuitem">
+            <a href="/careers">Careers</a>
+          </li>
+        </ul>
+      </noscript>
+    </nav>
+  </div>
+  {% endblock header_banner %}
+</header>
+<div class="dropdown-window-overlay fade-animation"></div>
+<div class="dropdown-window slide-animation">
+  <div class="u-hide" id="products-content"></div>
+  <div class="u-hide" id="partners-content">  </div>
+  <div class="u-hide" id="careers-content"></div>
+</div>
+
+<script>
+  var nav = document.querySelector('.js-show-nav');
+  var hash = window.location.hash;
+  nav.classList.remove('u-hide');
+
+  // If the page loads with a preselected hash load and open the menu
+  if (hash) {
+    var selected = nav.querySelector(hash);
+    if (selected) {
+      selected.onmouseover();
+    }
+  }
+
+  function fetchDropdown(url, id) {
+    const div = document.getElementById(id);
+    fetch(url)
+    .then(function(response) {
+      return response.text();
+    })
+    .then(function(text) {
+      div.innerHTML = text;
+    })
+    .catch(function(error) {
+      console.log('Request failed', error)
+    })
+  }
+</script>
+
+<script>
```

Comment:
Any reason why you don't put all the JS in one `<script>` tag instead of splinting it in 2?

Reply:
They are separate solutions to different things. Stops things clashing and makes it easier to pull them apart if we want in the future.